sql/core/src/main/java/org/apache/spark/sql/sources/v2/TableProvider.java

```diff
@@ -24,7 +24,7 @@
 
 /**
  * The base interface for v2 data sources which don't have a real catalog. Implementations must
- * have a public, 0-arg constructor.
+ * have a public, 0-arg constructor, and can only deal with existing tables.
```

Comment:
The phrase "deal with" is not very specific. I think it would be better to say what these tables can be used for: data operations like read, append, delete, and overwrite. Not operations that require metadata changes.